lca-cli/ops/execute.go

```diff
@@ -116,17 +128,10 @@ func NewChrootExecutor(logger *logrus.Logger, verbose bool, root string) Execute
 	return &chrootExecutor{executor: executor{logger, verbose}, root: root}
 }
 
-func (e *chrootExecutor) baseExecute(writer io.Writer, command string, args ...string) (string, error) {
-	commandBase := "/usr/bin/env"
-	args = append([]string{command}, args...)
-	arguments := []string{"--", "bash", "-c", strings.Join(args, " ")}
-	return e.executor.execute(writer, e.root, commandBase, arguments...)
-}
-
 func (e *chrootExecutor) Execute(command string, args ...string) (string, error) {
-	return e.baseExecute(nil, command, args...)
+	return e.executor.executeBash(nil, e.root, command, args...)
```

Comment:
(continuing previous comment) Actually, I didn't realize all of our executions are always through bash. This is not good. We should change that

Reply:
Created #258

Reply:
I think originally the bash execs were for the chroot'ed unshare "remount and run command" steps, and maybe ended up as the default exec when the executor code was created?

Reply:
We need the `/bin/env` for chroot, but I can get rid of the `bash -c` here, and instead only use it where absolutely necessary. Also adding a comment to explain the need for the `env`